utils/utils.go

```diff
@@ -82,10 +88,10 @@ func getHomeDir() string {
 	return homedir
 }
 
-func expand(path string) string {
+func Expand(path string) string {
```

Comment:
If you already here and it's made public, I think `ExpandPath` is better.  

Reading `utils.Expand`  in place of use, some time after it was used for a first time is totally out of context

Reply:
Or even ExpandHome

Reply:
what about more precise ExpandHomePath?

Reply:
sure